testsuite/features/reposync/srv_wait_for_reposync.feature

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019-2022 SUSE LLC
+# Copyright (c) 2019-2023 SUSE LLC
```

Comment:
Why do you modify the date? You did not touch anything else here?

Reply:
I reverted some changes there and forgot to revert the date, thanks for pointing that out!